portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+function addRandomLanguage() {
+  const languages =
+      ['Español', 'English', 'Deutsch'];
```

Comment:
Wow that's so fun that you know German! You don't actually have to change anything, just wanted to say that was cool haha

Reply:
Jaja thanks Gloria! Perhaps one day I can teach you the basics.